README.md

```diff
@@ -153,20 +187,20 @@ meta.tofile('example.sigmf-meta') # extension is optional
 
 ## Frequently Asked Questions
 
-#### Is this a GNU Radio effort?
+### Is this a GNU Radio effort?
 
 *No*, this is not a GNU Radio-specific effort. It is hosted under the GNU Radio
 Github account because this effort first emerged from a group of GNU Radio core
 developers, but the goal of the project to provide a standard that will be
 useful to anyone and everyone, regardless of tool or workflow.
 
-#### Is this specific to wireless communications?
+### Is this specific to wireless communications?
 
 *No*, similar to the response, above, the goal is to create something that is
 generally applicable to _signal processing_, regardless of whether or not the
 application is communications related.
 
-#### It seems like some issues take a long time to resolve?
+### It seems like some issues take a long time to resolve?
```

Comment:
I agree that the `###` is more appropriate and looks better.

Reply:
Yeah I imagined having a TOC one day, want subsections to make sense

Reply:
👍🏼